Library/Homebrew/hardware.rb

```diff
@@ -214,6 +214,24 @@ def oldest_cpu(_version = nil)
       end
     end
     alias generic_oldest_cpu oldest_cpu
+
+    # Returns a Rust flag to set the target CPU if necessary.
+    # Defaults to nil.
+    sig { returns(T.nilable(String)) }
+    def rustflags_target_cpu
+      # Rust already defaults to the oldest supported cpu for each target-triple
+      # so it's safe to ignore generic archs such as :armv6, etc., here.
+      # Rust defaults to apple-m1 since Rust 1.71 for aarch64-apple-darwin.
```

Comment:
```suggestion
      # Rust already defaults to the oldest supported cpu for each target-triplet
      # so it's safe to ignore generic archs such as :armv6 here.
      # Rust defaults to apple-m1 since Rust 1.71 for aarch64-apple-darwin.
```